Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTableView.cpp

```diff
@@ -29,7 +29,7 @@ namespace AzToolsFramework
     {
         AssetBrowserTableView::AssetBrowserTableView(QWidget* parent)
             : QTableView(parent)
```

Comment:
You should be able to get this for free by just changing this to inherit from AzQtComponents::TableView

Reply:
That's pretty neat! Just added it and is making my life much easier. I'm playing around with it right now to see what else can I do. Thanks!